integration-tests/test_display_name_option.py

```diff
@@ -133,3 +133,59 @@ def test_invalid_display_name(invalid_display_name, insights_client):
     assert (
         origin_display_name == record["display_name"]
     ), "display-name should remain unchanged when new display-name is rejected"
+
+
+def test_display_name_disable_autoconfig_and_autoupdate(insights_client, test_config):
+    """
+    This is a satellite bug(https://issues.redhat.com/browse/ESSNTL-2230),
```

Comment:
(nit) to maintain uniformity in all tests, we should replace it with something like : 
Ref :  https://issues.redhat.com/browse/ESSNTL-2230,

Reply:
@Archana-PandeyM you mean name this case as "Hilti config + Satellite"? But @ptoscano suggested "please NEVER mention customer names in public stuff", and that's why I choose current test case name.

Reply:
by name I meant my name, I see already removed now. Thanks!
No I would also suggest not to put any customer specific data in upstream tests. You might want to edit the comment and remove names form above comment as well.
What I am suggesting is to replace line 140  with "Ref : https://issues.redhat.com/browse/ESSNTL-2230"

Reply:
LGTM, please sqash commit messages into one. Rebase branch.